.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Run Smoke Tests'
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+  workflow_dispatch:
+jobs:
+  smoke_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install operating system dependencies
+        run: npx playwright install-deps
```

Comment:
We will need caching and pinning for the browser versions in order to speed up the set up time. We want smoke tests to run as fast as possible after a deployment (preferably within a minute). If we want to we could even have a slow run for known browser versions and a fast run for the pinned browser versions.

Perhaps we can have a Docker image with everything already set up in order to take off sooner.

<img width="654" alt="image" src="https://user-images.githubusercontent.com/44410/130797252-6bcb3bea-20a7-48c8-a091-d787262f284f.png">

<img width="460" alt="image" src="https://user-images.githubusercontent.com/44410/130797315-88d308bd-74f3-41ee-a7ce-50c60aabc67c.png">

Do the tests run against all 4 browsers?
Does it say anywhere against what version of each?